account_revoke_token.php

```diff
@@ -0,0 +1,48 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * This page stores the reported bug
+ *
+ * @package MantisBT
+ * @copyright Copyright 2000 - 2002  Kenzaburo Ito - kenito@300baud.org
+ * @copyright Copyright 2002  MantisBT Team - mantisbt-dev@lists.sourceforge.net
+ * @link http://www.mantisbt.org
+ *
+ * @uses authentication_api.php
+ * @uses core.php
+ * @uses helper_api.php
+ * @uses print_api.php
+ * @uses user_api.php
+ */
+
+require_once( 'core.php' );
+require_api( 'authentication_api.php' );
+require_api( 'helper_api.php' );
+require_api( 'print_api.php' );
+require_api( 'user_api.php' );
+
```

Comment:
Prevent from being called directly by something like
if( !defined( 'ACCOUNT_PREFS_REVOKE_TOKEN_ALLOW' ) ) {
	return;
}
define ACCOUNT_PREFS_REVOKE_TOKEN_ALLOW in account_page.php